engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/analyse/alias/Graph.scala

```diff
@@ -24,11 +27,26 @@ sealed class Graph extends Serializable {
     val copy = new Graph
     copy.rootScope     = this.rootScope.deepCopy(scope_mapping)
     copy.links         = this.links
+    copy.sourceLinks   = this.sourceLinks
+    copy.targetLinks   = this.targetLinks
     copy.globalSymbols = this.globalSymbols
     copy.nextIdCounter = this.nextIdCounter
     copy
   }
 
+  def initLinks(links: Set[Graph.Link]): Unit = {
```

Comment:
Shouldn't `initLinks` also populate `getLinks`? I don't see such behavior and I believe that calling:
```scala
val set = Set(l1, l2, l3);
g.initLinks();
assert g.getLinks() == set
```
would fail.